hoomd/md/minimize/fire.py

```diff
@@ -123,12 +150,44 @@ class FIRE(_DynamicIntegrator):
         quantity).
 
     Note:
-        As a default setting, the algorithm will start with a :math:`\\delta t
-        = \\frac{1}{10} \\delta t_{max}` and attempts at least 10 search steps.
-        In practice, it was found that this prevents the simulation from making
+        In practice, the default parameters prevents the simulation from making
         too aggressive a first step, but also from quitting before having found
-        a good search direction. The minimum number of attempts can be set by
-        the user.
+        a good search direction. Adjust the parameters as needed for your
+        simulations.
+
+    Attributes:
```

Comment:
`methods`, `forces`, `constraints`, and `rigid` don't show on the documentation.